src/App.jsx

```diff
@@ -13,58 +15,63 @@ export const goods = [
   'Jam',
   'Garlic',
 ];
+export const App = () => {
+  const goodStart = goods.find(value => value === 'Jam');
+  const [good, setGood] = useState(goodStart);
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {good ? (
+          <>
+            {`${good} is selected`}
             <button
-              data-cy="RemoveButton"
+              onClick={() => {
+                setGood(null);
+              }}
```

Comment:
Create a helper function reset for this logic and use it everywhere
```suggestion
              onClick={reset}
```